src/lib.rs

```diff
@@ -50,21 +49,28 @@
 //! }
 //! ```
 //!
-//! [`reqwest`](reqwest#proxies) supports system proxies by
-//! default. It reads the environment variables `HTTP_PROXY` and `HTTPS_PROXY`
-//! environmental variables to set HTTP and HTTPS proxies, respectively.
+//! ### Proxies
+//!
+//! [`reqwest`](reqwest#proxies) supports system proxies by default. It reads
+//! the environment variables `HTTP_PROXY` and `HTTPS_PROXY` environmental
+//! variables to set HTTP and HTTPS proxies, respectively.
+//!
+//! ### Environmental variables
 //!
-//! Rspotify supports the [`dotenv` crate](dotenv), which allows you to save
-//! credentials in a `.env` file. These will then be available as environmental
-//! values when using methods like [`CredentialsBuilder::from_env`](crate::oauth2::CredentialsBuilder::from_env):
+//! Rspotify supports the [`dotenv`] crate, which allows you to save credentials
```

Comment:
Recover the reference linked to `dotenv` crate?

Reply:
What?

```
[`dotenv`] 
```

Is enough for an intra-doc link.